data/config/__init__.py

```diff
@@ -62,6 +63,8 @@ def __init__(self, filename_base, width, height):
         self.time_format = json["time_format"]
         self.end_of_day = json["end_of_day"]
         self.full_team_names = json["full_team_names"]
+        self.use_team_abbrev = json["use_team_abbrev"]
```

Comment:
You'd want to add this to config.json.example (also, consider making it part of the layout rather than the config)